src/vfcwrapper/main.c

```diff
@@ -51,25 +51,45 @@
 #define CALL_OP_SIZE 4
 #endif
 
-typedef double double2 __attribute__((ext_vector_type(2)));
-typedef double double4 __attribute__((ext_vector_type(4)));
-typedef double double8 __attribute__((ext_vector_type(8)));
-typedef double double16 __attribute__((ext_vector_type(16)));
-typedef float float2 __attribute__((ext_vector_type(2)));
-typedef float float4 __attribute__((ext_vector_type(4)));
-typedef float float8 __attribute__((ext_vector_type(8)));
-typedef float float16 __attribute__((ext_vector_type(16)));
-typedef int int2 __attribute__((ext_vector_type(2)));
-typedef int int4 __attribute__((ext_vector_type(4)));
-typedef int int8 __attribute__((ext_vector_type(8)));
-typedef int int16 __attribute__((ext_vector_type(16)));
+// typedef int int2 __attribute__((ext_vector_type(2)));
+// typedef int int4 __attribute__((ext_vector_type(4)));
+// typedef int int8 __attribute__((ext_vector_type(8)));
+// typedef int int16 __attribute__((ext_vector_type(16)));
+
+// typedef float float2 __attribute__((ext_vector_type(2)));
+// typedef float float4 __attribute__((ext_vector_type(4)));
+// typedef float float8 __attribute__((ext_vector_type(8)));
+// typedef float float16 __attribute__((ext_vector_type(16)));
+
+// typedef double double2 __attribute__((ext_vector_type(2)));
+// typedef double double4 __attribute__((ext_vector_type(4)));
+// typedef double double8 __attribute__((ext_vector_type(8)));
+// typedef double double16 __attribute__((ext_vector_type(16)));
+
```

Comment:
Can we remove these comment lines ?
As we demand to user to use only `clang` or `gcc` and `clang` using `OpenCL` or `gcc` vectors types.
We might as well use `gcc` vectors.
Because we don't use `OpenCL` tricks like `v.xyz`.

Reply:
Right, I forgot to remove these lines. Actually I'll change for `ext_vector_type` which is the OpenCL extension.